### ISSUE TITLE
[Android] Digimon story cyber sleuth not starting when running with NCE

### ISSUE THREAD

**Issue by Controlvalve** [yuzu-emu/yuzu#12149]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Android - 140

### Description of Issue

Game won't start when running with NCE. 
Other setting: default
Driver: Turnip R-X

### Expected Behavior

Game to start

### Reproduction Steps

Start game with NCE on

### Log File

> [yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/13459629/yuzu_log.txt.old.txt)

### System Configuration

CPU: 
GPU/Driver: Adreno 740
RAM: 16 gb
OS: Android 14

**Comment by Controlvalve** [comment]
![Screenshot_20231125_085001_org yuzu yuzu_emu](https://github.com/yuzu-emu/yuzu/assets/151936444/b404239d-50c7-4df5-a6f1-9ebac77cbc70)

**Comment by liamwhite** [comment]
Game works fine for me - check your dump

**Comment by Controlvalve** [comment]
Mine work fine when running with JIT. Problem only exist when run with NCE. 
Is dump possibly the problem here? It work fine with NCE with build 139 and previous ver.

**Comment by liamwhite** [comment]
Provide a log when running with JIT

**Comment by xUju19** [comment]
Mine was like this too when updating to 141, but when I close/stop the Yuzu app and delete the previous shader it's starting to works again.

**Comment by Controlvalve** [comment]
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/13462957/yuzu_log.txt)

I have no problem when running with JIT (attached log when start game with JIT)

**Comment by Controlvalve** [comment]
Tested delete shader, reset setting, start again with NCE on. It show building shader for 20 minutes before i exit the game. 
![Screenshot_20231125_094702_org yuzu yuzu_emu](https://github.com/yuzu-emu/yuzu/assets/151936444/b2702213-ae84-4a0b-bf29-b2ccf721d3a4)

**Comment by xUju19** [comment]
Yeah, it's kinda getting harder to get inside the game now for me too.
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/13465777/yuzu_log.txt)

**Comment by xUju19** [comment]
> [yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/13462957/yuzu_log.txt)
> 
> I have no problem when running with JIT (attached log when start game with JIT)

Try running the game with fastmem off and backend Cpu NCE, because when fastmem on sometimes the game doesn't start(stuck in loading) and sometimes it start properly, but with fastmem off, the game always start.
![Screenshot_2023-11-29-12-12-50-511_org yuzu yuzu_emu](https://github.com/yuzu-emu/yuzu/assets/151410572/7ef8c9b6-b7ac-4658-8e25-b571a0208916)
